config/v3_5_experimental/types/schema.go

```diff
@@ -195,11 +195,12 @@ type Proxy struct {
 }
 
 type Raid struct {
-	Devices []Device     `json:"devices,omitempty"`
-	Level   *string      `json:"level,omitempty"`
-	Name    string       `json:"name"`
-	Options []RaidOption `json:"options,omitempty"`
-	Spares  *int         `json:"spares,omitempty"`
+	Assemble *bool        `json:"assemble,omitempty"`
```

Comment:
Since you added a struct field you need to add a translation for it in translations, this is why CI is failing.